ios/RNSScreenStackHeaderConfig.m

```diff
@@ -467,6 +468,59 @@ + (void)updateViewController:(UIViewController *)vc withConfig:(RNSScreenStackHe
   }
 }
 
++ (UIBlurEffectStyle)blurEffectStyleFromEffectName:(NSString *)effectName
+{
+  if (@available(iOS 13.0, *)) {
+    if ([effectName  isEqual: @"extraLight"]) {
```

Comment:
Could this be a `RCT_ENUM_CONVERTER`?

Reply:
Yeah, I will probably change it when I'll have some time.